### PR TITLE
new update

### PR DESCRIPTION
**Pull request by jyoti-wp**
Updated on feature branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # klinic-project
+An Example of html project.
```
Comment (imranhsayed) [inline README.md:2]: @jyoti-wp 
Please remove extra dot

**Review by imranhsayed — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jyoti-wp) [reply README.md:2]: Thanks for the update

Comment (imranhsayed) [reply README.md:2]: Your welcome

File: Klinic-hospital_files/magnific-popup.css [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+  right: 0;
+  top: 0;
+  text-decoration: none;
+  text-align: center;
+  opacity: 0.65;
+  padding: 0 0 18px 10px;
+  color: #FFF;
+  font-style: normal;
+  font-size: 28px;
+  font-family: Arial, Baskerville, monospace; }
```
Comment (imranhsayed) [inline Klinic-hospital_files/magnific-popup.css:139]: Please give a  line below

File: html/index.html [hunk 3]
```diff
@@ -1,17 +1,337 @@
...
+				<button>Eng</button>
+				<button>My Account</button>
+			</div>
+		</div>
+	</div>
+</div>
+<div class="wrapper">
+	<div class="banner-container">
+		<div class="preloader">
+			<img src="images/logo.png" alt="">
```
Comment (imranhsayed) [inline html/index.html:41]: Please add alt text for accessibility for disable people like so :
`Logo`

File: html/index.html [hunk 4]
```diff
@@ -1,17 +1,337 @@
...
+		</div>
+		<div class="menu">
+			<ul>
+				<li>
+					<a class="" href="">HOME </a>
```
Comment (imranhsayed) [inline html/index.html:46]: Remove extra space after Home.
Change the HOME to lowercase letters and use `text-transform:uppercase` to make it uppercase using css.

If there is not class name , remove `class=""`

File: html/index.html [hunk 5]
```diff
@@ -1,17 +1,337 @@
...
+		</li>
+		<li>
+			<a>About us</a>
+		</li>
+	</ul>
+</div>
+<div class="media-content-section">
+	<div class="media-content-left">
+		<a href="https://www.youtube.com/watch?v=GT6-H4BRyqQ" class="video-popup" target="_blank">
+			<img src="images/play-button.png" alt="play button"></a>
```
Comment (imranhsayed) [inline html/index.html:88]: Please fix the indentation ( tabs and new line)

File: html/index.html [hunk 6]
```diff
@@ -1,17 +1,337 @@
...
+
+</div>
+<div class="skill-area">
+	<div class="wrapper">
+		<div class="image-container">
+			<div class="image-section">
+				<img src="images/1.jpg" alt="">
+				<img src="images/2.jpg" alt="">
+				<img src="images/3.jpg" alt="">
+				<img src="images/4.jpg" alt="">
```
Comment (imranhsayed) [inline html/index.html:134]: @jyoti-wp `alt` values  are missing

File: html/index.html [hunk 7]
```diff
@@ -1,17 +1,337 @@
...
+	</div>
+</div>
+<div class="team-area-section">
+	<div class="team-area">
+		<div class="our-experts">
+			<h3>Our expert doctors</h3>
+		</div>
+		<div class="teams">
+			<div class="team-one">
+				<img src="images/1.png" alt="">
```
Comment (imranhsayed) [inline html/index.html:170]: alt tag?

File: html/index.html [hunk 8]
```diff
@@ -1,17 +1,337 @@
...
+</div>
+
+<div class="signup-section">
+	<h5>Signup for Our Newslatter</h5>
+
+	<div class="newsletter-area">
+		<form action="#" class="wow fadeInUp" data-wow-delay="0.4s">
+			<input type="email" placeholder="Enter Your Email Address" required="">
+			<button type="submit" class="primary-light-bg"><i class="fa fa-paper-plane"></i></button>
+		</form>
```
Comment (imranhsayed) [inline html/index.html:240]: `</button>` will go on a new line.
Please fix indentation

File: html/index.html [hunk 9]
```diff
@@ -1,17 +1,337 @@
...
+<div class="latest-update-section">
+	<div class="wrapper">
+		<div class="latest-update-section-one">
+			<h2>Latest Update</h2>
+			<p>Condimentum rutrum placerat egestas condimentum mi eros. Eleifend cras quirntum</br>Feugiat elit
+				placerat. Diam tempor malesuada</p>
+		</div>
+		<div class="latest-update-section-two">
+			<div class="admin-section">
+				<img src="images/1 (1).jpg" alt="admin">
```
Comment (imranhsayed) [inline html/index.html:253]: image names should be separated using '-'
e.g. 
`admin`
Should be :
`admin`

File: html/index.html [hunk 10]
```diff
@@ -1,17 +1,337 @@
...
+                            <span>
+                                <a href>doctor</a>
+
+                                <a href>Clinic</a>
+                            </span>
+						</div>
+					</div>
+                    <h2>Remember Outweigh Do He Desirous No Cheerful</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore</p>
+                    <a href="" class="bttn-mid btn-emt">Read More</a>
```
Comment (imranhsayed) [inline html/index.html:271]: `href` value is missing.
Please put the value of `href` e.g.
` <a href="http://google.com" class="bttn-mid btn-emt">Read More</a>`

**Review by imranhsayed — CHANGES_REQUESTED** [review]

**Pull request merged**